lib/ansible/utils/display.py

```diff
@@ -201,11 +201,13 @@ def debug(self, msg, host=None):
                 self.display("%6d %0.5f [%s]: %s" % (os.getpid(), time.time(), host, msg), color=C.COLOR_DEBUG)
 
     def verbose(self, msg, host=None, caplevel=2):
```

Comment:
Would it be useful to allow `verbose` to accept `stderr` in the function signature too?  We would also have to update `v`, `vv`, etc to support it also.

Reply:
yep, but thought that would be huge undertaking while this was a 'blanket' requriement